cpu/native/async_read.c

```diff
@@ -42,6 +42,8 @@ static void _async_io_isr(void) {
 
     int max_fd = 0;
 
+    struct timeval timeout = { .tv_usec = 0 };
```

Comment:
>> +    struct timeval timeout = { .tv_usec = 0 };
> 
> Does this set |timeout.tv_sec|, too?

Yes, static initializers null unset values implicitly. |{0}| would be
equivalent here.